.github/workflows/smoke-checks.yml

```diff
@@ -22,68 +22,67 @@ concurrency:
 
 env:
   HOMEBREW_NO_INSTALL_CLEANUP: 1 # Disable cleanup for homebrew, we don't need it on CI
-  XCODE_VERSION: 13.4.1
-  IOS_VERSION: "15.5"
-  IOS_SIMULATOR_DEVICE: "iPhone 12 Pro"
-  IOS_SIMULATOR_DEVICE_WITH_VERSION: "iPhone 12 Pro (15.5)"
+  XCODE_VERSION: "14.0.1"
+  IOS_VERSION: "15.4"
+  IOS_SIMULATOR_DEVICE_WITH_VERSION: "iPhone 12 Pro (15.4)"
 
 jobs:
-  automated-code-review:
-    name: Automated Code Review
-    runs-on: macos-12
-    if: ${{ github.event_name != 'push' }}
-    steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/bootstrap
-      - name: Run Danger
-        run: bundle exec danger
-        env:
-          GITHUB_TOKEN: ${{ secrets.DANGER_GITHUB_API_TOKEN }}
-      - name: Run Linting
-        run: ./Scripts/run-linter.sh
+  # automated-code-review:
+  #   name: Automated Code Review
+  #   runs-on: macos-12
+  #   if: ${{ github.event_name != 'push' }}
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: ./.github/actions/bootstrap
+  #     - name: Run Danger
+  #       run: bundle exec danger
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.DANGER_GITHUB_API_TOKEN }}
+  #     - name: Run Linting
+  #       run: ./Scripts/run-linter.sh
```

Comment:
Are all these flows meant to be disabled?

Reply:
Nah, just debugging failing snapshot test on CI